api/user/User.java

```diff
@@ -0,0 +1,10 @@
+package com.vaticle.typedb.client.api.user;
+
+import javax.annotation.CheckReturnValue;
+
+public interface User {
+    @CheckReturnValue
+    String name();
```

Comment:
The API does not support updating password yet. It will be added on the subsequent PR.